tests/integration/targets/snap_alias/tasks/test.yml

```diff
@@ -0,0 +1,47 @@
+---
+- name: Ensure snap 'hello-world' is not installed
+  community.general.snap:
+    name: hello-world
+    state: absent
+
+- name: Ensure snap 'hello-world' is installed now
+  community.general.snap:
+    name: hello-world
+
+- name: Create snap alias
+  community.general.snap_alias:
+    name: hello-world
+    alias: hw
+  register: alias_single
```

Comment:
How about also testing check mode and idempotency? I.e. replace this one task by four ones (check mode, regular, check mode idempotency, regular idempotency). (Same then for the other tasks.)

Reply:
supporting check mode would be a good start :P

Reply:
In that case, you should mention in `notes` that check_mode is not supported.

Reply:
(And idempotency checks still make sense :) )

Reply:
done